website/docs/r/virtual_network.html.markdown

```diff
@@ -105,6 +109,12 @@ The following attributes are exported:
 
 * `address_space` - The address space that is used the virtual network.
 
+* `subnet` - Set of multiple subnets defined in this virtual network. Each `subnet` block supports the fields documented below.
+
+The `subnet` block supports:
```

Comment:
can we add `---` above this to add a divider as we're doing for the new blocks, and also change `supports` to `exports`(since the values aren't setable)